### PR TITLE
Allows wine to execute bat and msi files

### PR DESCRIPTION
**Pull request by qparis**
Allows wine to execute bat and msi files

### CODE REVIEW

File: Functions/Engines/Wine/script.js [hunk 1]
```diff
@@ -177,6 +177,16 @@ Wine.prototype.run = function (executable, args, captureOutput) {
         args = [];
     }
 
+    var extensionFile = executable.split(".").pop();
+
+    if(extensionFile == "msi") {
+        return this.run("msiexec", ["/i", executable], captureOutput);
```
Comment (plata) [inline Functions/Engines/Wine/script.js]: As I said in another comment already, I'm not sure this is really a good idea. You loose completely the ability to run a msi with anything else than `/i`.

Comment (qparis) [inline Functions/Engines/Wine/script.js]: You could still use msi command.
I think I need to add the arguments

Comment (plata) [inline Functions/Engines/Wine/script.js]: True (because msiexec does not have the extension "msi"). Do you want me to merge?

Comment (qparis) [inline Functions/Engines/Wine/script.js]: Yes this can be merged if you are ok

**Pull request merged**